core/src/main/java/com/linecorp/armeria/client/HttpResponseDecoder.java

```diff
@@ -31,8 +31,8 @@ interface HttpResponseDecoder {
 
     InboundTrafficController inboundTrafficController();
 
-    HttpResponseWrapper addResponse(
-            int id, DecodedHttpResponse res, ClientRequestContext ctx, EventLoop eventLoop);
+    HttpResponseWrapper addResponse(AbstractHttpRequestHandler requestHandler, int id,
```

Comment:
Should we mark `requestHandler` as `@Nullable` and handle the `null`-case properly, given that we call this method with `null` in `HttpClientPipelineConfigurator`?